inst/_pkgdown.yml

```diff
@@ -193,7 +193,7 @@ reference:
       - onStop
       - diskCache
       - memoryCache
-      - reexports
+      - key_missing
```

Comment:
Shouldn't `key_missing` be in  `reexports`?

And should `reexports` still be here, or is it not necessary because of its functions have `@keywords internal` now?

Reply:
This is for the website only.  For the website, we should link to `fastmap::key_missing`.

(`reexports` has been removed)

Reply:
Before this PR, `reexports` was just `fastmap::key_missing`